keps/sig-apps/3329-retriable-and-non-retriable-failures/README.md

```diff
@@ -1579,6 +2044,15 @@ For each of them, fill in the following information by copying the below templat
 
 ###### What steps should be taken if SLOs are not being met to determine the problem?
 
+If pods terminate without the expected pod failure conditions:
```

Comment:
These look like useful troubleshooting in general. The SLO you define is specifically around the job sync duration. Will these help debug that? It's not clear to me.

Reply:
I think using the SLO metrics can be useful. Added two points based on them.